collectors/python.d.plugin/pandas/pandas.chart.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# Description: pandas netdata python.d module
+# Author: Andrew Maguire (andrewm4894)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import pandas as pd
+import requests
```

Comment:
Can use `try/except` here and log a message in check() (better then generic exception message)

e.g

```python
try:
    import X
    HAS_X = True
except ImportError:
    HAS_X = False
...

   def check(self):
       if not HAS_X:
           ...
```

Reply:
added warning if it cant be imported